includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -67,6 +67,15 @@ private static function get_primary_configuration() {
 		return null;
 	}
 
+	/**
+	 * Get the parent configuration ID.
+	 *
+	 * @return string|null
+	 */
+	public static function get_parent_configuration_id() {
+		return self::get_primary_configuration()->parent ?? self::TEST_MODE_CONFIGURATION_PARENT_ID;
```

Comment:
Maybe we can return null instead of the test mode constant when there is no PMC... so we can show a "configuration error" message.

Reply:
Oh right, makes sense. Updated it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4219/commits/8a823d86327505c9083e47e7c57df539c13433da